### PR TITLE
Add theme_color member (closes #225)

### LINKED ISSUE #225: A means to control the "theme color"
When an application is launched in standalone mode (or any mode that shows a status bar), it would be nice to be able to have a way of controlling the color of the status bar. 

Proprietary examples:
* IE' [mapplication-navbutton-color](http://msdn.microsoft.com/en-us/library/gg491732%28v=vs.85%29.aspx#msapplication-navbutton-color), 
* iOS's [apple-mobile-web-app-status-bar-style](https://developer.apple.com/library/safari/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/MetaTags.html)

See also [Chromium's Intent to Implement and ship: brand-color meta tag](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/nzRY-h_-_ig/KR3XWn73tDoJ)

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -127,7 +127,8 @@
   "scope": "/racer/",
   "start_url": "/racer/start.html",
   "display": "fullscreen",
-  "orientation": "landscape"
+  "orientation": "landscape",
+  "theme-color": "aliceblue"
```
Comment (benfrancis) [inline index.html]: typo: "theme-color" -> "theme_color"?

File: index.html [hunk 2]
```diff
@@ -1422,6 +1423,77 @@ <h3 id="applying">
...
+        <p>
+          The <dfn id="member-theme_color"><code>theme_color</code></dfn>
+          member serves as the <dfn>default theme color</dfn> for an
+          application context. What constitutes a <dfn>theme color</dfn> is
+          defined in [[!META-THEME-COLOR]].
+        </p>
+        <p>
+          If the user agent honors the value of the <code>theme_color</code>
+          member as the <a>default theme color</a>, then that color serves as
+          the <a>theme color</a> for all browsing context to which the manifest
```
Comment (benfrancis) [inline index.html]: "context" -> "contexts"?

File: index.html [hunk 3]
```diff
@@ -1422,6 +1423,77 @@ <h3 id="applying">
...
+          For all intents and purposes, the <a><code>theme_color</code></a>
+          member is functionally equivalent to having a <a><code>meta</code>
+          element</a> whose <a title="name attribute">name</a> attribute is
+          <a><code>theme_color</code></a> in a [[!HTML]] document. In cases
+          where no suitable theme can be derived from processing a manifest, a
+          [[!META-THEME-COLOR]] can serve as a suitable fallback.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>theme_color</code>
+          member</dfn> is given by the following algorithm. The algorithm takes
```
Comment (benfrancis) [inline index.html]: "is" -> "are"?

Comment (marcoscaceres) [inline index.html]: Fixed typos, thanks Ben!

**Comment by slightlyoff** [comment]
Love this. Thank you!

**Pull request merged**